spec/page_spec.rb

```diff
@@ -339,7 +377,7 @@ class PageWithParameterizedUrlMatcher < SitePrism::Page
 
   describe 'with a regexp matcher' do
     class PageWithRegexpUrlMatcher < SitePrism::Page
-      set_url_matcher(/foos\/(\d+)/)
+      set_url_matcher(%r{foos\/(\d+)})
```

Comment:
I actually disagree with this cop and ban it from other suites. Can we revert this and just have it as an override